src/web3/networkConfig.ts

```diff
@@ -66,6 +66,14 @@ interface NetworkConfig {
     [key in CiCoCurrency | Currency]: string
   }
   celoTokenAddress: string
+  alchemyEthereumRpcUrl: string
+}
+
+const ALCHEMY_ETHEREUM_RPC_URL_STAGING = 'https://eth-sepolia.g.alchemy.com/v2/'
+const ALCHEMY_ETHEREUM_RPC_URL_MAINNET = 'https://eth-mainnet.g.alchemy.com/v2/'
+
+export type blockExplorerUrls = {
```

Comment:
kinda weird to me how there is a lowercase-b version of this type and an uppercase-B version, which are similar but a little different